pkg/volume/csi/csiv0/csi.pb.go

```diff
@@ -1482,13 +1482,13 @@ type TopologyRequirement struct {
 	// If requisite is specified, all topologies in preferred list MUST
 	// also be present in the list of requisite topologies.
 	//
-	// If the SP is unable to to make the provisioned volume available
+	// If the SP is unable to make the provisioned volume available
```

Comment:
@saad-ali @msau42 are these copied directly from upstream CSI? If so, we probably don't want to mess with them

Reply:
Correct, we had to copy the csi spec protos directly here because of issues when trying to vendor multiple csi spec versions.

Reply:
Roll back the previous version?

Reply:
Yes, revert this file